main.go

```diff
@@ -164,14 +156,8 @@ func generateThumbnail(w http.ResponseWriter, rmethod, rpath string, sourceURL s
 	defer resp.Body.Close()
 
 	if resp.StatusCode != 200 {
-		copyHeader(w.Header(), resp.Header)
-		io.Copy(w, resp.Body)
-		return
-	}
-
-	contentType := resp.Header.Get("Content-Type")
-	if !strings.HasPrefix(contentType, "image/") {
-		http.Error(w, fmt.Sprintf("invalid content type %q", contentType), 500)
```

Comment:
we had some images that did not have a proper content-type stored on the file on S3. VIPS will read the first couple of bytes anyway to try to determine if it's a PNG or JPEG.

Example: https://mls-crawler-results-production.s3.amazonaws.com/residential_photos/4335448/20100212012930854903000000-o.jpg_